src/psycop/common/model_evaluation/binary/time/test_timedelta_data.py

```diff
@@ -37,7 +37,7 @@ def test_get_timedelta_series():
 
 def test_sensitivity_by_timedelta():
     df = str_to_df(
-        """t1_timestamp,t2_timestamp,y,pred,
+        """t1_timestamp,t2_timestamp,y,y_hat_probs,
```

Comment:
```suggestion
        """t1_timestamp,t2_timestamp,y,y_hat,
```

For sensitivity, we need the prediction, not the score :-)